checks/containers/container_engine/xccl_tests.py

```diff
@@ -85,36 +84,53 @@ def set_perf(self):
                 self.stdout, 'gbs', float)
         }
 
+class XCCLTestsBaseCE(XCCLTestsBase, ContainerEngineMixin):
+    container_env_table = {
+        'annotations.com.hooks': {
+            'aws_ofi_nccl.enabled': 'true'
+        }
+    }
 
-@rfm.simple_test
-class NCCLTestsCE(XCCLTestBase):
-    descr = 'Point-to-Point and All-Reduce NCCL tests with CE'
-    valid_systems = ['+ce +nvgpu']
-    image_tag = parameter(['cuda12.9.1'])
+    @run_before('run')
+    def set_pmix(self):
+        self.job.launcher.options += ['--mpi=pmix']
 
-    # Disable Nvidia Driver requirement
-    env_vars['NVIDIA_DISABLE_REQUIRE'] = 1
+        # Disable MCA components to avoid warnings
+        self.env_vars.update(
+            {
+                'PMIX_MCA_psec': '^munge',
+                'PMIX_MCA_gds': '^shmem2'
+            }
+        )
 
-    # Disable MCA components to avoid warnings
+def _set_xccl_uenv_env_vars(env_vars):
     env_vars.update(
         {
-            'PMIX_MCA_psec': '^munge',
-            'PMIX_MCA_gds': '^shmem2'
+            'NCCL_CROSS_NIC': '1',
+            'NCCL_NET': '\'AWS Libfabric\'',
+            'NCCL_NET_GDR_LEVEL': 'PHB',
+            'NCCL_PROTO': '^LL128',
+            'FI_CXI_DEFAULT_CQ_SIZE': '131072',
+            'FI_CXI_DEFAULT_TX_SIZE': '32768',
+            'FI_CXI_DISABLE_HOST_REGISTER': '1',
+            'FI_CXI_RDZV_EAGER_SIZE': '0',
+            'FI_CXI_RDZV_GET_MIN': '0',
+            'FI_CXI_RDZV_PROTO': 'alt_read',
+            'FI_CXI_RDZV_THRESHOLD': '0',
+            'FI_CXI_RX_MATCH_MODE': 'software',
+            'FI_MR_CACHE_MONITOR': 'userfaultfd',
         }
     )
 
-    reference_per_test = {
-        'sendrecv': {
-            '*': {
-                'GB/s': (24.0, -0.05, None, 'GB/s')
-            }
-         },
-        'all_reduce': {
-            '*': {
-                'GB/s': (75.0, -0.05, None, 'GB/s')
```

Comment:
Note that I removed this (lower) expected performance for rccl and use the 90 GB/s expected for nccl for both. On one and two nodes the expected performance is anyway higher than 90 or 100 GB/s and the reference needs to reflect that eventually. This is also for a follow-up PR though.